bin/sync_slurm_acct.py

```diff
@@ -85,13 +87,24 @@ def main():
     opts = ExtendedSimpleOption(options)
     stats = {}
 
+    last_timestamp = opts.options.start_timestamp
+    if not last_timestamp:
+        try:
+            last_timestamp = read_timestamp(SYNC_TIMESTAMP_FILENAME)
+        except Exception:
+            _log.warning("Something broke reading the timestamp from %s", SYNC_TIMESTAMP_FILENAME)
+            last_timestamp = "201710230000Z"
+            _log.warning("We will resync from a hardcoded know working sync a while back : %s", last_timestamp)
+
+    _log.info("Using timestamp %s", last_timestamp)
+    # record starttime before starting, and take a 10 sec safety buffer so we don't get gaps where users are approved
+    # in between the requesting of modified users and writing out the start time
+    start_time = datetime.datetime.now(tz=utc) + datetime.timedelta(seconds=-10)
+    _log.info("startime %s", start_time)
+
```

Comment:
isn't this block of code common to all 'sync' script? Maybe factor it out to a common place?

Reply:
i also agree to move this block and the one with the writer to vsc.utils.timestamp

Reply:
k.

Reply:
Proposal is in https://github.com/hpcugent/vsc-utils/pull/55, so I could add it. If it's approved.